### DISCUSSION TITLE
Dall E

### DISCUSSION THREAD

**Discussion by JohnessyAI** [srcnalt/OpenAI-Unity#60] [General]
I was just wandering if this system uses the upto date dall e as when im using the sample scene the images generated are not that great? I need to say again mate your Package is absolutely amazing!

**Comment by srcnalt** [comment]
Hi @JohnessyAI, thank you! Packages makes direct requests to [DallE endpoint](https://platform.openai.com/docs/api-reference/images) with out any post or preprocessing. I assume improving prompt quality might play a role in receiving better image results.

**Reply by JohnessyAI** [reply]
Thanks for this mate! Is it using Dall e or Dalle 2? Im hoping for something like mid journey quality. If its not what would be the starting point to start making those changes? Thank you lad

**Reply by srcnalt** [reply]
I did notice low quality of results too when I was working on the tool, increasing the size of the image to 1024 helps with that and certain prompts do better than others. You can try `cartoon style cat` and get horrendous result but `anime style cat` yields much better. The docs say Image API still is in beta so guessing it's not the best they provide just yet. And yes Midjourney is much ahead of DallE at this point.

**Reply by JohnessyAI** [reply]
You are a legend sarge! Thank you!

**Reply by JohnessyAI** [reply]
Hey bud, Quick thing, is there anyway i can contact you or ask you a question about the tutorial you did using polly?

**Reply by srcnalt** [reply]
Sure, feel free to ask under [Smart NPC Project](https://github.com/srcnalt/RPM-Smart-NPC) discussions, which is where Polly was used in my tutorials.
